vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -53,528 +88,1345 @@ The code below shows the steps necessary to add a box plot to the page:
 
-As you can see from the code, `first_page` is added to the [`Dashboard`][vizro.models.Dashboard] and the dashboard is displayed by running `Vizro().build(dashboard).run()`.
+## 3. Create a second page
 
-### 2.2. Add further components
+### 3.1. Add a chart
 
-You can combine and arrange various types of `components` on a dashboard page. The `components` currently available are [`Card`][vizro.models.Card], [`Graph`][vizro.models.Graph], and [`Button`][vizro.models.Button]. For more information, refer to the [components](../user-guides/components.md) overview page to find the guide for each type.
+Next, you'll learn how to add a second page to the dashboard, featuring charts and KPI (Key Performance Indicator) cards.
 
-The code below adds two components to the page:
+Vizro uses [`Graph`][vizro.models.Graph] models and [Plotly Express functions](https://plotly.com/python/plotly-express/) to create various types of charts. You can explore some of the available chart types and their code examples in the [Vizro visual vocabulary](https://vizro-demo-visual-vocabulary.hf.space).
 
-- A [`Card`][vizro.models.Card] to insert markdown text into the dashboard.
-- A [`Graph`][vizro.models.Graph] to illustrate GDP development per continent since 1952 as a line graph.
+Follow these steps to add a histogram to the page:
 
-!!! warning "Before you run this code in a Jupyter Notebook"
-    If you are following this tutorial in a Jupyter Notebook, you need to restart the kernel each time you evaluate the code. If you do not, you will see error messages such as "Components must uniquely map..." because those components already exist from the previous evaluation.
+1. Create a second [`Page`][vizro.models.Page] and store it in a variable called `second_page`. Set its `title` to `"Summary"`.
+1. Add a [`Graph`][vizro.models.Graph] to the `components` list.
+1. Inside the `figure` argument of the `Graph`, use the code for the [px.histogram from the visual vocabulary](https://vizro-demo-visual-vocabulary.hf.space/distribution/histogram).
+1. Add the new page to the list of `pages` in the [`Dashboard`][vizro.models.Dashboard] by calling `vm.Dashboard(pages=[first_page, second_page])`.
 
-!!! example "Add components"
-    === "Code first component"
+!!! example "Second Page"
+    === "Snippet - Second Page"
         ```py
+        second_page = vm.Page(
+            title="Summary",
+            components=[
+                vm.Graph(figure=px.histogram(tips, x="total_bill")),
+                vm.Graph(figure=px.histogram(tips, x="tip")),
+            ],
+        )
+        dashboard = vm.Dashboard(pages=[first_page, second_page])
+        ```
 
-        vm.Card(
-            text="""
-                # First dashboard page
-                This pages shows the inclusion of markdown text in a page and how components
-                can be structured using Layout.
-            """,
+    === "Code - Dashboard"
+        ```{.python pycafe-link hl_lines="22-28 30"}
```

Comment:
I'm not finding this renders on PyCafe. @maxschulz-COL did we find a way around passing parameters to the mkdocs plugin?